index.rst

```diff
@@ -77,19 +85,39 @@ categorized as:
 Tag Fragments
 -------------
 
-In general, tags can be postfixed with an optional Cycle number or a
-``rest`` set of one-or-more underscore-separated fields.  The Cycle is
-currently unique to T&S RSP instances, and specifies the version of XML
-used in this image.  It will have the form ``c<digits>.<digits>`` where
-the first group of digits is the cycle number and the second group of
-digits is the build iteration for that cycle, e.g. ``c0019.001``.
+Primary
+^^^^^^^
+The primary version of a tag is derived from (and in most cases identical to) the EUPS tag of the underlying DM stack included in the image.
+It is always the first component of the tag.
+
+RSP Build Counter
+^^^^^^^^^^^^^^^^^
+
+Tags may include an identifier specifying an RSP image build count. 
+In general, any run of the GitHub Action that builds a sciplat-lab image will increment this counter.
```

Comment:
In the separate implementation section, this should link to the GitHub documentation for the specific variable being used and its semantics.

There's also a lot of other stuff that we talked about that I think should be captured in that implementation section: the other alternatives considered (a simple build number, a semver), the intent of this number (to be able to index into release notes so that people know if capabilities are available), and the intent to assign this number specifically to rebuilds with the same Pipelines release version.